docs/source/index.md

```diff
@@ -0,0 +1,73 @@
+% batconf documentation entrypoint
+
+```{currentmodule} batconf
+```
+
+```{toctree}
+:caption: Introduction
+:hidden: true
+:maxdepth: 2
+
+Welcome to BatConf <self>
+Intro <intro>
+Quick Start Guide <quickstart>
+Dev Guide <devguide>
+```
+
+```{toctree}
+:caption: Reference
+:hidden: true
+:maxdepth: 6
+
+API <reference/modules>
+```
+
+# BatConf
+
+> *Configuration Management for Python projects, modules, applications, 
+> and microservices*
+
+[![Stable Version](https://img.shields.io/pypi/v/batconf?color=blue)](https://pypi.org/project/batconf/)
+[![Build Status](https://github.com/lundybernard/batconf/actions/workflows/tests.yml/badge.svg)](https://github.com/lundybernard/batconf/actions)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/batconf)
+[![Downloads](https://img.shields.io/pypi/dm/batconf)](https://pypistats.org/packages/batconf)
+
+Unify your python application and module configuration.
+Allow users in different contexts to use the config source that works best for
+them.  Set reasonable defaults, and override them as needed. 
+
+* Hierarchical priority: CLI > Environment > config file > module defaults
+* Provides builtin support common config sources:
```

Comment:
Provides builtin support *for* common...

Reply:
fixed